formatTest/typeCheckedTests/expected_output/mlVariants.re

```diff
@@ -4,9 +4,9 @@ type polyVariantsInMl = [
   | `StillAnIntTuple(int, int)
 ];
 
-let intTuple = `IntTuple(1, 2);
+let intTuple = `IntTuple((1, 2));
```

Comment:
Can this double parens be avoided?

Reply:
Yes, and what about `OrdinaryVariant((1, 2))` & `myFunctionApplication((1, 2))` ?

Reply:
Oh wait, that can't be avoided righr

Reply:
How did this pass previously?

Reply:
Nvm, see my comment